lib/openc_bot/helpers/register_methods.rb

```diff
@@ -159,8 +159,8 @@ def schema_name
 
       def stale_entry_uids(stale_count=nil)
         stale_count ||= default_stale_count
-        sql_query = "ocdata.#{ primary_key_name } from ocdata WHERE retrieved_at IS NULL OR strftime('%s', retrieved_at) < strftime('%s',  '#{Date.today - days_till_stale}') LIMIT #{stale_count.to_i}"
-        select(sql_query).each do |res|
+        entries = select("ocdata.#{ primary_key_name } from ocdata WHERE retrieved_at IS NULL OR strftime('%s', retrieved_at) < strftime('%s',  '#{Date.today - days_till_stale}') order by retrieved_at is null asc, datetime(retrieved_at) asc LIMIT #{stale_count.to_i}")
```

Comment:
Why are we changing from using iterator functionality of select? Also this breaks specs, so we need to change those too.